src/database/seeders/20240411132319-add-user.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
           email: 'test4@example.com',
           phone: '1234567890',
           verified: true,
-          // roleId: 'd290f1ee-6c54-4b01-90e6-d701748f0851',
+           roleId: 'd290f1ee-6c54-4b01-90e6-d701748f0852',
```

Comment:
why changing this test I know this is not your implementation

Reply:
how can you use user without a roleId??

Reply:
i only added role id to that user in seeders so test that use this user can work